reference/forms/twig_reference.rst

```diff
@@ -217,6 +217,28 @@ array).
 
     <option {% if choice is selectedchoice(value) %} selected="selected"{% endif %} ...>
 
+.. _form-twig-rootform:
+
+rootform
+~~~~~~~~
+
+This test will check if the current ``form`` does not have a parent form view.
+
+.. code-block:: twig
+
+    {# DON'T DO THIS: this simple check can't differentiate between a form having
+       a parent form view and a form defining a normal form field called 'parent' #}
```

Comment:
```diff
- normal
+ nested
```
?

Reply:
fixed in 640e8e56763ffa3084d27482863672cbc9caea4a